tractor/conda-science_demo.yml

```diff
@@ -0,0 +1,37 @@
+name: science_demo
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.11
+  - numpy
+  - pandas
+  - scipy
+  - pyarrow
+  - swig
+  - ipykernel
+  - cairo
+  - cython
+  - "ceres-solver<2.2.0"
+  - pkg-config
+  - setuptools
+  - wcslib
+  - libjpeg-turbo
+  - netpbm
+  - acstools
+  - astropy
+  - astroquery
```

Comment:
pyvo is pulled in by astroquery, but it would good to spell it out separately as there is direct use of it. 

Maybe the policy could be to list everything that is directly imported in the demo notebooks?